java/src/main/native/src/row_conversion.cu

```diff
@@ -304,20 +334,600 @@ copy_from_fixed_width_columns(const cudf::size_type start_row, const cudf::size_
   }
 }
 
+#if !defined(__CUDA_ARCH__) || __CUDA_ARCH__ >= 700
+
+/**
+ * @brief The GPU blocks work on one or more block_info structs of data.
+ *        This structure defined the workspace for the block.
+ *
+ */
+struct block_info {
```

Comment:
"block" is probably a bad term here.  Overlaps too strongly with cuda blocks.

Reply:
The `_optimized` code path seems to prefer the term `row_group`. This is the "tile" that's processed per grid-block?

Reply:
Changed to tile nomenclature for these.